apps/image-server/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "aws-sdk": "^2.1003.0",
+    "cors": "^2.8.5",
```

Comment:
Waar gebruikte we in dit geval cors voor?

Reply:
Die gebruikte we omdat we cors aan wilde zetten voor connectie met de image server van buitenaf, maar dit is uiteindelijk ongebruikt gebleven omdat we een andere manier vonden. Is er nu uit gehaald.